### PR TITLE
Updates of some repo URLs that changed, some licenses and slight corrections of original spellings

### PR DESCRIPTION
**Pull request by Trilarion**
Corrected some original spellings (see their Wikipedia pages for verification): AquaStax -> Aquastax, Mega Lo Mania -> Mega-Lo-Mania, Outrun -> Out Run and Simcity to SimCity. Also corrected within each game entry where these originals were referenced.

Corrected some licenses, are GPL3 but were marked GPL2 (may have been upgraded in between): see http://svn.code.sf.net/p/lgames/code/trunk/lbreakouthd/COPYING and http://svn.code.sf.net/p/rescue/code/Rescue/gpl.txt

And updated some URLs (repos mostly) that have changed. Simply follow the old URLs to see that they were outdated.

I haven't changed the 'updated' field. Would do that after the PR has been reviewed.

### CODE REVIEW

File: games/0.yaml [hunk 1]
```diff
@@ -71,8 +71,8 @@
   originals:
   - Runescape Classic
   type: remake
-  repo: https://github.com/2006rebotted/2006rebotted
-  url: https://rsrebotted.com/
+  repo: https://github.com/2006-Scape/2006Scape
```
Comment (cxong) [inline games/0.yaml:74]: need to rename this game to 2006Scape too

File: games/d.yaml [hunk 2]
```diff
@@ -132,7 +132,7 @@
   development: active
   originals:
   - Quake
-  repo: https://svn.icculus.org/twilight/trunk/darkplaces/
+  repo: https://gitlab.com/xonotic/darkplaces.git
```
Comment (cxong) [inline games/d.yaml:135]:
```suggestion
  repo: https://gitlab.com/xonotic/darkplaces
```

File: games/h.yaml [hunk 3]
```diff
@@ -86,7 +86,7 @@
   development: active
   originals:
   - Story of Seasons series
-  repo: https://github.com/greentwip/HarvestMoon
+  repo: https://github.com/Greentwip/HoneyTown
```
Comment (cxong) [inline games/h.yaml:89]: also rename to Honey Town

File: games/l.yaml [hunk 4]
```diff
@@ -82,7 +82,7 @@
   development: active
```
Comment (cxong) [inline games/l.yaml:82]:
```suggestion
  development: halted
```
let's update this as well; repo is archived

File: games/o.yaml [hunk 5]
```diff
@@ -1778,8 +1778,8 @@
   - WebGL
   license:
   - GPL3
-  url: https://github.com/rage8885/OpenSC2K
-  repo: https://github.com/rage8885/OpenSC2K
+  url: https://github.com/nicholas-ochoa/OpenSC2K
```
Comment (cxong) [inline games/o.yaml:1781]:
```suggestion
```
repo and url are the same, so we don't need both

File: games/r.yaml [hunk 6]
```diff
@@ -406,7 +406,7 @@
   lang:
   - Java
   license:
-  - GPL2
+  - GPL3
```
Comment (cxong) [inline games/r.yaml:409]: please also change the repo to https://sourceforge.net/projects/rescue/

**Review by cxong — CHANGES_REQUESTED** [review]

File: games/0.yaml (hunk 1 above)
Comment (Trilarion) [reply games/0.yaml:74]: Sure

File: games/h.yaml (hunk 3 above)
Comment (Trilarion) [reply games/h.yaml:89]: sure

File: games/r.yaml (hunk 6 above)
Comment (Trilarion) [reply games/r.yaml:409]: Sure

**Comment by Trilarion** [comment]
Thanks for looking over it.

**Review by cxong — APPROVED** [review]

**Comment by Trilarion** [comment]
Updated 'updated' fields (except for very small changes like trailing slash or name change of original) and changed url field of Gift Grabber (gave 404 before). From my side ready to merge.

**Pull request merged**